README.md

```diff
@@ -361,6 +361,7 @@ The aims is to provide a ready-to-run recipes that you can just copy, paste and
  - [Miniflux](rss/../apps/rss/miniflux.md)
  - [Miniflux-filter](rss/../apps/rss/miniflux-filter.md)
  - [FreshRSS](https://www.freshrss.org/) 🔗 - second best :-)
+ - [RSS to telegram bot](apps/rss/rrstt.md)
```

Comment:
missed one :-)

Reply:
On God I forgot to change back the readme link 

I mean have some patience
I'm kinda new to github
:-)

Reply:
no worries. 
I code review for living :-)